src/utils/Logger.cpp

```diff
@@ -29,6 +31,7 @@ Logger::Logger(logLevel level, std::string prefix) :  _log_level(level), _msg_lo
 	filenamestream << ".log";
 	_filename = filenamestream.str();
 	_log_stream = new std::ofstream(_filename.c_str());
+	*_log_stream << std::boolalpha;  // Print boolean as true/false
 }
```

Comment:
Changes to the logger should go to a separate commit/PR IMHO.

Reply:
Maybe for now, it is a bit pointless to move it to a separate commit as I would have to delete it in a first commit and add it again in a second one. A PR would be a bit over the top for one line...
But in general, you are right, I will consider it next time.